src/Domain/CoordinateMaps/Wedge3D.hpp

```diff
@@ -240,11 +240,17 @@ class Wedge3D {
    * somewhere in between
    * \param with_equiangular_map Determines whether to apply a tangent function
    * mapping to the logical coordinates (for `true`) or not (for `false`).
-   * \param halves_to_use Determines whether to use the logical xi
-   * coordinates in the [0,1] interval (value of `UpperOnly`) of the full wedge,
-   * the coordinates in the [-1,0] interval (value of `LowerOnly`) of the
-   * full wedge, or the full wedge entirely (value of `Both`). Half wedges are
-   * currently only useful in constructing domains for binary systems.
+   * \param halves_to_use Determines whether to split the Wedge3D in half. If
+   * the wedge is split in half, the resulting shape has a face normal to the x
+   * direction (assuming default OrientationMap). If the wedge is split in half,
+   * an intermediate affine map is applied to the logical xi coordinate such
+   * that the interval [-1,1] is mapped to the corresponding logical half of the
+   * wedge. For example, if `UpperOnly` is specified, [-1,1] is mapped to
+   * [0,1], and if `LowerOnly` is specified, [-1,1] is mapped to [-1,0]. In the
+   * case of `Both`, no intermediate map is applied. In all cases, the logical
+   * points returned from the inverse map function call lie in the range [-1,1]
```

Comment:
Minor: please reword to "...points returned by the inverse map will lie in the range..."